### ISSUE TITLE
Bypassing a Real Censorship Scenario: HELP

### ISSUE THREAD

**Issue by h-saber** [sshuttle/sshuttle#559]
Dear friends, 

I live in an extremely internet-censored area which sometimes the big-brother decides to block ALL connections to the internet for an unknown duration and routes all the traffics in local intranet.

This is the technical case:
1. There rarely exist some special servers who "always" have access to the outside (for example the backend server of a local service application which uses Google Maps) but these servers' traffic are highly monitored and investigated by the cops.
2. I have access to some of these special servers who can access the free world, but their access is censored, limited and under control. Let's call them "in-cage vps"
3. I have some other servers in the real free world which I can go through them to free internet. Let's call them "free vps"

Now my plan:
A) Runnig some VPN server (like WG) on in-cage vps.
B) Tunneling the whole traffic of in-cage vps to free vps in an encrypted way. (obfuscation is a plus, if available).
C) So my home laptop or mobile phone can access the free world through these two server hubs.

After a few search I found sshuttle a great tool for the tunneling part between two servers.

BUT:
If I have already started my vpn from my laptop to in-cage vps, and then start sshuttle, everything works fine. But this can't be the normal usage! 
The problem is when I first start sshuttle in in-cage vps, the vps won't accept any new incoming connection and so noone can use it, except those sessions which are generated before starting of sshuttle like my current ssh terminal.
How can I handle this issue?
Any helping solution is appropriated.

**Comment by svyotov** [comment]
I have used successfully in the past stunnel + openvpn. Stunnel can create end to end encrypted tls tunnel over port 443.
Traffic to the outside world looks like any normal encrypted webpage connection (except the amount of data) and then you can do your openvpn or any other connection from inside. https://github.com/Xaqron/stunnel (or google stunnel + openvpn)

**Comment by h-saber** [comment]
Thanks @svyotov 
Currently I use the system you suggested but my question is different: 
I have one client and 2 servers.
When I install and run sshuttle on server1, it doesn't accept incoming connection from OpenVPN of the client.

**Comment by h-saber** [comment]
_UPDATE:_
I have installed some VPN servers on my in-cage VPS using **Streisand**, and installed **sshuttle 1.0.5** using git clone. **But when I connect to those VPN servers, the outgoing connection won't go through sshuttle** (I tried WG, OpenVPN and SSH Tunnel)
How can I handle this issue?
Any helping solution is appropriated.

**Comment by svyotov** [comment]
shot in the dark, but this does not seem like a VPN issue, aare you sure it is not related to https://github.com/sshuttle/sshuttle/issues/554 ? What is your OS/ DNS server?

**Comment by h-saber** [comment]
Nope. It's Ubuntu 16 and sshuttle works perfect, except I can't route the VPN servers' traffic through it.
I've even tried this instruction but no success:
https://www.reddit.com/r/WireGuard/comments/i2hkhq/using_wireguard_with_sshuttle/

**Comment by ali80** [comment]
> Nope. It's Ubuntu 16 and sshuttle works perfect, except I can't route the VPN servers' traffic through it. I've even tried this instruction but no success: https://www.reddit.com/r/WireGuard/comments/i2hkhq/using_wireguard_with_sshuttle/

have you found any solutions for this?
I have the same issue, when using sshuttle, I connect from the `cage vps` to `free vps`, the `cage vps` stops receiving requests from other clients, 
so in the other words, the connection between `cage vps` and `free vps` is ok, but when I try to connect to `cage vps` from my laptop, it fails, I cant even establish ssh connection anymore
